### PR TITLE
networkctl: VXLan - display more properties

### PR DESCRIPTION
**Pull request by ssahani**

### CODE REVIEW

File: src/network/networkctl.c [hunk 1]
```diff
@@ -1539,6 +1554,8 @@ static int link_status_one(
                         return table_log_add_error(r);
 
         } else if (streq_ptr(info->netdev_kind, "vxlan")) {
+                char str[MAX(strlen("auto") + 1, DECIMAL_STR_MAX(uint8_t))];
```
Comment (poettering) [inline src/network/networkctl.c]: gcc sucks and doesn't grok that strlen() is constant, still turns this into a VLA. please use STRLEN() here, i.e. the uppercase macro we defined which is guaranteed to be constant

**Comment by ssahani** [comment]
Updated . Thanks for the review . PTAL

File: src/network/networkctl.c [hunk 2]
```diff
@@ -1539,6 +1554,8 @@ static int link_status_one(
                         return table_log_add_error(r);
 
         } else if (streq_ptr(info->netdev_kind, "vxlan")) {
+                char str[MAX(STRLEN("auto") + 1, DECIMAL_STR_MAX(uint8_t))];
```
Comment (poettering) [inline src/network/networkctl.c]: CONST_MAX (so that this doesn't become a variable-length array)

**Comment by ssahani** [comment]
Updated. Thanks for the review.  PTAL .

**Pull request merged**